apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V27__Create_Enrollmenttable.sql

```diff
@@ -0,0 +1,141 @@
+/*
+ * The column doesn't have a default value to avoid updating the column on migration. The pipeline server
+ * will populate the column as new beneficaries are added or existing beneficaries are updated. 
+ */
+
+${logic.tablespaces-escape} SET default_tablespace = fhirdb_ts2;
+
+create table "Enrollments" (
+    "parentBeneficiary" varchar(255) not null,
+    "yearMonth" varchar(7) not null,
```

Comment:
Why this column type instead of an actual date type? I'd think an actual date type would make it easier to query against and work with, though I haven't thought about it too hard.

I did some quick research, which you should absolutely double-check me on, and it looks like the general recommendation here is to use a `date` column with the day component pinned to `1`.